src/Neo/SmartContract/Native/PolicyContract.cs

```diff
@@ -615,6 +633,47 @@ private StorageIterator GetBlockedAccounts(DataCache snapshot)
             return new StorageIterator(enumerator, 1, options);
         }
 
+        [ContractMethod(Hardfork.HF_Faun, CpuFee = 1 << 15, RequiredCallFlags = CallFlags.States | CallFlags.AllowNotify)]
+        internal async ContractTask<bool> RecoverFunds(ApplicationEngine engine, UInt160 account, UInt160 token)
+        {
+            var committeeMultiSigAddr = AssertAlmostFullCommittee(engine);
+
+            // Set request time
+
+            var key = CreateStorageKey(Prefix_BlockedAccount, account);
+            var entry = engine.SnapshotCache.GetAndChange(key, null)
+                ?? throw new InvalidOperationException("Request not found.");
+            if (engine.GetTime() - (BigInteger)entry < RequiredTimeForRecoverFunds)
+                throw new InvalidOperationException("Request must be signed at least 1 year ago.");
```

Comment:
Better to know how much time remain.
```suggestion
            var elapsedTime = engine.GetTime() - (BigInteger)entry;
            if (elapsedTime < RequiredTimeForRecoverFunds)
            {
                var remaining = (BigInteger)RequiredTimeForRecoverFunds - elapsedTime;
                var days = remaining / 86_400_000;
                var hours = (remaining % 86_400_000) / 3_600_000;
                var minutes = (remaining % 3_600_000) / 60_000;
                var seconds = (remaining % 60_000) / 1_000;
                var timeMsg = days > 0 ? $"{days}d {hours}h {minutes}m"
                    : hours > 0 ? $"{hours}h {minutes}m {seconds}s"
                    : minutes > 0 ? $"{minutes}m {seconds}s"
                    : $"{seconds}s";
                throw new InvalidOperationException($"Request must be signed at least 1 year ago. Remaining time: {timeMsg}.");
            }
```

<img width="1091" height="241" alt="28b7ced1-a100-4377-9c5c-f9099410f9f6" src="https://github.com/user-attachments/assets/65210b8f-7ad0-4b1b-a9dd-e451fbce959b" />